js/components/sortable-densities.js

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from 'react'
+import {SortableContainer, SortableElement, arrayMove} from 'react-sortable-hoc'
+import { hsv } from 'd3-hsv'
+
+import '../../css/sortable-densities.less'
+
+let key = 0
+
+function hsvToBackground (hsvColor) {
+  let rgb = hsv(hsvColor.h, hsvColor.s, hsvColor.v).rgb()
+  return {backgroundColor: 'rgb(' + Math.floor(rgb.r) + ', ' + Math.floor(rgb.g) + ', ' + Math.floor(rgb.b) + ')'}
+}
+
+const SortableItem = SortableElement(({plateInfo}) =>
+  <li className='density-button-container' style={hsvToBackground(plateInfo.color)}>
+    <div className='shading-box'>
+      <div className='density-button' key={key++}> {plateInfo.label} </div>
```

Comment:
1. I think `key` should be always applied to the top element. I've checked console and react is complaining about missing keys.
2. I'd say that assigning keys like this is against the purpose of the `key` property. It should be used to identify which elements has changed and can be updated, which need to be removed and which need to be added to DOM. Here, it's pretty similar to saying `key=Math.random()`. React will remove and re-add all the elements on each render pass. It's not a big here, as we have 5 buttons usually. But in general, this approach might cause issues e.g. with performance and animations. So, I believe it's good to keep it right. Key should identify an element in a unique way, so react knows what to do with its DOM. The obvious candidate here would be `plateInfo.id`.

Reply:
Done, and good point! I'll keep that in mind for the future.